admin/helpers/sce.js

```diff
@@ -0,0 +1,27 @@
+'use stict'
+var moment = require('moment-timezone')
+const tzMetaData = require('moment-timezone/data/meta/latest.json')
+
+const getOffset = zone => {
+  return moment().tz(zone).format('Z')
+}
+
+const scePresenter = {}
+
+scePresenter.getCountriesTzData = () => {
+  const countryZones = []
+  Object.values(tzMetaData.countries).forEach(val => {
+    if (val.zones.length > 1) {
+      val.zones.forEach(z => {
+        const zoneName = z.split('/').pop().replace('_', ' ')
+        countryZones.push({ name: `${val.name}, ${zoneName} (GMT ${getOffset(z)})`, zone: z })
+      })
+    } else {
+      const z = val.zones[0]
+      countryZones.push({ name: `${val.name} (GMT ${getOffset(z)})`, zone: z })
+    }
+  })
+  return countryZones
```

Comment:
This looks like it might be worth caching.  The `json` file will only be updated by a release, and so the parsed `countryZones` data should be good for the entire life of the app.

On the other hand, if this is only used by the service-manager, it won't make much difference.

Reply:
That is true indeed, it can be cached - I didn't think of it because the json data is not that huge in size. Caching added! 👍